README.md

```diff
@@ -36,16 +36,31 @@ The documentation within this repo assumes `CLI tools` are being used and
 the setup has been completed, specifically for
 [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
 
+## Testing
+
+Run the unit tests with the command `dotnet test`, from the root dir that
+would be `dotnet test ./Tests`. The tests can be run continuously using a file
+watcher via `dotnet watch test`, however, for this to work the command must be
+issued from within the `./Tests` dir.
+
 ## Running Locally
 
-To start the function app run `func start`. Once the function is running it can
-be triggered by running [run-function-locally](./scripts/run-function-locally)
-and passing the name of the function i.e.
-`./scripts/run-function-locally ExtractAWData`.
+The function can be started by running `func start` from within
+`./ExtractAWData`. Prior to running the command a valid
+`local.settings.json` must exist in the function dir.
 
 `local.settings.json` will need to contain the relevant secrets for
 the function app to work correctly. Along with the `.p12` certificate file
-with no password must also be available in the repo's root directory.
+with no password must also be available in the function dir.
+
+Once the function is running it can be triggered by running
+[run-function-locally](./scripts/run-function-locally) from the root dir
+and passing the name of the function i.e.
+`./scripts/run-function-locally ExtractAWData`.
```

Comment:
Do you think it's worth adding a comment for windows users that the script requires bash commands? One way of running it outside of WSL would be via Git Bash.

It hasn't been stated on the other Node.js GWA projects, but this being a C# project may meant it could be picked up by a windows developer to support.

Reply:
Sure, will do.